docs/examples/pytorch/single_stage_detector/pytorch_ssd.rst

```diff
@@ -0,0 +1,76 @@
+Single Shot MultiBox Detector training in PyTorch
+============================
+
+This example implements Single Shot Multibox Detector originally published by Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy, 
```

Comment:
Pleaae stress out the purpose - to show how DALI works in the detection networks, not how to implement SSD itself. It will be included in DALI docs so it should be alligned with it.

Reply:
Done